.github/workflows/release.yml

```diff
@@ -64,6 +64,21 @@ jobs:
             - name: Set name
               run: git config user.name "Zowe Robot"
 
+            - name: Release to NPM automatic
+              run: |
+                  cd onboarding-enabler-nodejs
+                  echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > ~/.npmrc
+                  echo "registry=$DIST_REGISTRY" >> ~/.npmrc
+                  npm version ${{ github.event.inputs.scope || env.DEFAULT_SCOPE }}
+                  npm publish --access public
+                  git add package.json
```

Comment:
Why is there only add?

Reply:
Commit happens as a part of the release. So the idea is to have just one commit.